bedrock/externalpages/templates/externalpages/pocket/pocket-and-firefox.html

```diff
@@ -0,0 +1,102 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block pocket_css %}
+  {{ css_bundle('pocket-features') }}
+{% endblock  %}
+
+{% block page_title %}Pocket for Firefox{% endblock %}
```

Comment:
**Thought**: I wonder if we should change the pocket title that ends with `| Pocket`? SInce this page would end up being `Pocket for Firefox | Pocket` out of scope for this, but just a thought.